src/encoded/static/components/motifs.js

```diff
@@ -189,14 +189,14 @@ export class MotifElement extends React.Component {
 
         const targetListLabel = (targetList.indexOf(',') !== -1) ? 'Targets' : 'Target';
         const pwmsLabel = (pwmsLength > 1 || pwmsLength === 0) ? 'PWMs' : 'PWM';
-        const footprintsLabel = (footprintsLength > 1 || footprintsLength === 0) ? 'Footprints' : 'Footprint';
+        const footprintsLabel = (footprintsLength > 1 || footprintsLength === 0) ? `Footprints (${footprintsLength})` : 'Footprint';
         
         const targetComponent = (
             <React.Fragment>
             {targetList.length > 0 && (
-                <React.Fragment>
-                    {targetListLabel} {targetList}
-                </React.Fragment>
+                <div className='label-value-pair'>
+                    {targetListLabel} &nbsp; <div className='value'>{targetList}</div>
```

Comment:
It’s not wrong, but I personally prefer to use margins instead of non-breaking spaces to space elements. Since `targetListLabel` isn’t in its own div or span, I realize my way gets a little awkward.